src/components/content/Content.component.scss

```
@@ -0,0 +1,12 @@
+@import "src/assets/styles/_variables.scss";
+
+.content {
+  padding: 0.625rem 12.5rem;
+  &__button__create {
```

Please follow BEM structure properly
```suggestion
  padding: 0.625rem 12.5rem;
  &__button--create {
```